File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Ugyldiggjoere.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class Ugyldiggjoere {
    private InvalideringSed sed;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Ugyldiggjoere.kt
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class Ugyldiggjoere {
    private InvalideringSed sed;
}
